src/main/java/duke/task/Deadline.java

```diff
@@ -0,0 +1,30 @@
+package duke.task;
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+
+/**
+ * Represents a Deadline that is part of the Task class, regarding a deadline.
+ */
+
+public class Deadline extends Task {
+    protected LocalDate date;
+
+    /**
+     * The constructor to make a Deadline task.
+     * @param description the specific details of the task
+     * @param by the due date of the task
+     */
+    public Deadline(String description, String by) {
+        super(description);
+        this.date = LocalDate.parse(by);
+
```

Comment:
There's an extra blank line here before the closing brace as well. Should remove to follow coding standards.